core/tests/test_ingress.py

```diff
@@ -0,0 +1,237 @@
+"""
+Module to test importing data from sensors to db
+"""
+
+import os
+import pytest
+import pandas as pd
+
+from core.crop.db import create_database, connect_db, drop_db
+from core.crop.ingress import import_data
+from core.crop.ingress_adv import advanticsys_import
+from core.crop.populate_db import session_open, session_close, insert_advanticsys_data
+
+from core.crop.structure import TypeClass, LocationClass, SensorClass, SensorLocationClass
+
+from core.crop.constants import (
+    SQL_SERVER,
+    SQL_USER,
+    SQL_PASSWORD,
+    SQL_HOST,
+    SQL_PORT,
+    CONST_ADVANTICSYS_DIR,
+    CONST_ADVANTICSYS_TEST_1,
+    CONST_COREDATA_DIR,
+    CONST_TEST_DIR_DATA,
+    CONST_SENSOR_LOCATION_TESTS,
+    SQL_CONNECTION_STRING
+)
```

Comment:
Instead of adding **core.**, could you please adjust your python path since we have restructured the repository.